pkg/component/common/check_csi_install.go

```diff
@@ -0,0 +1,170 @@
+package common
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/kubeclipper/kubeclipper/pkg/utils/fileutil"
+	tmplutil "github.com/kubeclipper/kubeclipper/pkg/utils/template"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kubeclipper/kubeclipper/pkg/component"
+	"github.com/kubeclipper/kubeclipper/pkg/component/utils"
+	"github.com/kubeclipper/kubeclipper/pkg/logger"
+	v1 "github.com/kubeclipper/kubeclipper/pkg/scheme/core/v1"
+	"github.com/kubeclipper/kubeclipper/pkg/utils/cmdutil"
+	"github.com/kubeclipper/kubeclipper/pkg/utils/strutil"
+)
+
+func init() {
+	c := &CSIHealthCheck{}
+	if err := component.RegisterAgentStep(fmt.Sprintf(component.RegisterStepKeyFormat, CsiHealthCheck, version, HealthCheck), &CSIHealthCheck{}); err != nil {
```

Comment:
to `	if err := component.RegisterAgentStep(fmt.Sprintf(component.RegisterStepKeyFormat, CsiHealthCheck, version, HealthCheck), c); err != nil {
`